microsoft-edge/toc.yml

```diff
@@ -1033,89 +1050,166 @@
+                  href: extensions-chromium/publish/create-dev-account.md
+
+            - name: Register as an enterprise developer
+              items:
+#               - name: Register as a Microsoft Edge extension developer for Company Account
+#                 href:
+
+                - name: Verify your company account information
+                  href: extensions-chromium/publish/verify-microsoft-edge-program.md
 
-            - name: Verify your company account information
-              href: extensions-chromium/publish/verify-microsoft-edge-program.md
+                - name: Add users to the Microsoft Edge program
+                  href: extensions-chromium/publish/aad-account.md
 
             - name: Publish a Microsoft Edge extension by using a GitHub account
               href: extensions-chromium/publish/github.md
 
-            - name: Add users to the Microsoft Edge program
-              href: extensions-chromium/publish/aad-account.md
-
             - name: Publish a Microsoft Edge extension
               href: extensions-chromium/publish/publish-extension.md
 
-            - name: Extension hosting
-              href: extensions-chromium/publish/hosting-and-updating.md  # moved from /enterprise/
-
-            - name: Update a Microsoft Edge extension
-              href: extensions-chromium/publish/update-extension.md
+#           - name: Extension sharing offline
+#             href:
 
-            - name: Automatically update extensions in Microsoft Edge
-              href: extensions-chromium/publish/auto-update.md  # moved from /enterprise/
+            - name: Extension hosting
+              href: extensions-chromium/publish/hosting-and-updating.md
 
             - name: Submission states for extensions in the Microsoft Edge Add-ons website
               href: extensions-chromium/publish/submission-states.md
 
+            - name: Manage account settings
+              href: extensions-chromium/publish/manage-settings.md
+
+            - name: Alternative ways to distribute extensions
+              href: extensions-chromium/developer-guide/alternate-distribution-options.md
+
             - name: Curation and review process for the Edge Add-ons store
               href: extensions-chromium/publish/add-ons-store-curation.md
 
+            - name: REST API for publishing
+              items:
+                - name: Using the Microsoft Edge Add-ons REST API
+                  href: extensions-chromium/publish/api/using-addons-api.md
+
+                - name: REST API Reference for Microsoft Edge Add-ons
+                  href: extensions-chromium/publish/api/addons-api-reference.md
+
+# -------------------------------------
+        - name: Manage and maintain an extension
+          items:
+            - name: Update a Microsoft Edge extension
+              href: extensions-chromium/publish/update-extension.md
+
+            - name: Automatically update an extension in Microsoft Edge
```

Comment:
add to changelist at top of PR; old title:
Automatically update extensions in Microsoft Edge

Reply:
this was the particularly misrepresentative title, implying that a bulk operation is supported (it's not); 
Automatically update extensions in Microsoft Edge

the article body (the use-case; the user task) is actually about updating 1 extension.
@captainbrosset 
This fix changes the framing from abstract, system-oriented doc'n, to concrete, task-oriented doc'n.